public/locales/en/administration.json

```diff
@@ -200,5 +200,14 @@
         "whichSensor": "Which sensor would you like to download data for?",
         "chooseSensor": "Choose sensor",
         "anotherSensor": "Download data for another sensor"
+    },
+    "deleteSensor": {
+        "delete": "Delete Sensor",
+        "confirmDownload": "I have downloaded any data I need.",
+        "acknowledgeDelete": "I acknowledge that this action will delete any historical data for this sensor.",
+        "cannotBeUndone": "I acknowledge that this action cannot be undone",
+        "deleteNote": "Deleting a sensor will remove it from the map and delete all of its data. You will need to re-add the sensor to show this sensor on the map again and to collect this sensor's data. If you only want to temporarily stop showing this sensor or collecting data for this sensor, deactivate the sensor instead.",
```

Comment:
where does this appear in the code? I can't find it

Reply:
Oops good catch! I forgot to add it back when I changed to using a modal. I added it back above the sensor selector.